packages/api/src/controllers/sensorController.js

```diff
@@ -40,6 +40,19 @@ import { sensorErrors, parseSensorCsv } from '../../../shared/validation/sensorC
 import syncAsyncResponse from '../util/syncAsyncResponse.js';
 import knex from '../util/knex.js';
 
+const getSensorTranslations = async (language, res) => {
+  // Remove country identifier from language preference
+  const parsedLanguage = language.includes('-') ? language.split('-')[0] : language;
+  const translations = await import(`../../../shared/locales/${parsedLanguage}/sensorCSV.json`, {
+    assert: { type: 'json' },
+  });
+  // Handle case where user language not supported
+  if (!translations) {
+    res.status(422).send('User language not yet supported');
```

Comment:
On the front end side you defaulted to English if the language isn't supported, should we do the same here for consistency?

Reply:
Yeah maybe I overthought this one! 

I put a "TODO" below about the issue. We base the csv parser off of the users display language in the db. I was thinking it should really be passed as a param as part of the request body to state the language of the csv explicitly without relying on the users db value.

In any case, I was thinking if the the users db language is not supported -- can we assume the csv file they have attached is in english? 

I will put up a PR